helm/hpcc/values.yaml

```diff
@@ -444,6 +448,13 @@ esp:
     ## You can also specify labels on a service
     #labels:
     #  mylabel: "3"
+
```

Comment:
We don't need additional ingress rules at the individual service level. People can define a new visibility if they want different ingress rules for some services than others.

Reply:
little confused here. From previous comment I thought you suggest not define ingress rule in global.visibilities. So I plan to move the ingress definition here.